lib/rules/label-content-name-mismatch.json

```diff
@@ -0,0 +1,17 @@
+{
+  "id": "label-content-name-mismatch",
+  "matches": "label-content-name-mismatch-matches.js",
+  "tags": [
+    "wcag21a",
+    "wcag253"
+  ],
+  "metadata": {
+    "description": "Ensures that interactive elements labelled through their content must have their visible label as part of their accessible name",
+    "help": "Interactive elements must have their visible label as part of their accessible name"
```

Comment:
Name from content isn't just for interactive elements. I'm struggling to come up with a short but still descriptive text for this rule. Maybe talk to Jean or Carrie for suggestions?

Reply:
@jeankaplansky - can you help with a better help text here?

Reply:
Yep. I'm on it.

Reply:
@WilcoFiers , @jkodu : Is there a specific reason we cannot say "Ensures that ELEMENTS labeled through their content must have their visible label as part of their accessible name"? Ditto with the value for help: "ELEMENTS must have their visible label as part of their accessible name."

This approach accounts for all elements regardless of functional semantic intent (interactive or plain content).